repair_type/migrations/17.0.1.0.0/post-migration.py

```diff
@@ -10,10 +10,10 @@ def migrate(env, version):
         env.cr,
         """
         INSERT INTO stock_picking_type
-        (create_date, default_add_location_dest_id, default_remove_location_dest_id,
+        (create_date, default_location_dest_id, default_remove_location_dest_id,
```

Comment:
Yes, this script was created just when migrated to v17, but later https://github.com/OCA/repair/commit/c719218512a9f3b02f28a8e637004e61ba0ebc21 was merged.

Reply:
cc: @ppyczko